src/make_cbpp_profile.ado

```diff
@@ -0,0 +1,88 @@
+
+/***
+Title
+====== 
+
+__make_cbpp_profile__ {hline 2} Set up CBPP's standard profile.do.
+
+
+Description
+-----------
+
+__make_cbpp_profile__ retreives CBPP's standard [profile.do](https://www.stata.com/support/faqs/programming/profile-do-file/) and places it in the user's home directory. This command is only useful for CBPP staff. 
+
+CBPP's standard profile.do defines global macros that serve as shortcuts to synched SharePoint and OneDrive folders and cloned GitHub repositories. It also defines a global macro named 'censuskey' that contains the user's Census Bureau API key. Users may specify their API key in the _censuskey()_ option to  insert it into the profile.do. If this option is not specified, the 'censuskey' global will contain placeholder text.
+
+
+Syntax
+------ 
+
+> __make_cbpp_profile__, [_options_]
+
+{synoptset 27 tabbed}{...}
+{synopthdr}
+{synoptline}
+	{synopt:{opt censuskey}}Census Bureau API key.{p_end}
+	{synopt:{opt replace}}replace existing profile.do with standard profile.do.{p_end}
+
+
+Website
+-------
+
+[github.com/CenterOnBudget/cbppstatautils](https://github.com/CenterOnBudget/cbppstatautils)
+
+
+- - -
+{it:This help file was dynamically produced by {browse "http://www.haghish.com/markdoc/":MarkDoc Literate Programming package}.}
+***/
+
+
+* capture program drop make_cbpp_profile
+
+program define make_cbpp_profile
+
+	syntax, [censuskey(string)] [replace]
+	
+	
+	* set up cache and file paths ---------------------------------------------
+	
+	local cache_dir = cond("`c(os)'" == "Windows", 							///
+						   "~/AppData/Local/cbppstatautils",				///
+						   "~/Library/Application Support/cbppstatautils")
+	capture mkdir "`cache_dir'"	
+	
+	local remote_path "https://github.com/CenterOnBudget/cbpp-stata-utils/tree/master/ancillary"
```

Comment:
This is written to work once the PR is complete, right?

Reply:
Correct. You can un-comment the "debug" section (line 60) to test the command.